src/Native/Runtime.js

```diff
@@ -10,10 +10,15 @@ if (!Elm.fullscreen) {
 			NONE: 2
 		};
 
-		Elm.fullscreen = function(module, args)
+		Elm.fullscreen = function(module, args, connected)
 		{
+			if(typeof connected == 'undefined') {
+				connected = true;
+			}
 			var container = document.createElement('div');
-			document.body.appendChild(container);
+			if(connected) {
+				document.body.appendChild(container);
```

Comment:
TODO: think of a solution better than simply leaving this tree unconnected to the document. As is, there are 2 identical DOM trees (this one and the one owned by the debugger), so we incur DOM mutation costs twice.